app/src/main/java/com/grupo3/vinilos/album/list/AlbumsViewModel.kt

```diff
@@ -0,0 +1,34 @@
+package com.grupo3.vinilos.album.list
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.grupo3.vinilos.album.service.AlbumRepository
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.flow.asStateFlow
+import kotlinx.coroutines.flow.update
+import kotlinx.coroutines.launch
+import java.lang.Exception
+
+class AlbumsViewModel : ViewModel() {
+    private val repository = AlbumRepository()
+
+    private val _state = MutableStateFlow(AlbumsListState())
+    val state: StateFlow<AlbumsListState> = _state.asStateFlow()
+
+    fun getAlbums() {
+        viewModelScope.launch {
+            try {
+                val albums = repository.getAlbums()
+
+                _state.update { currentState ->
+                    currentState.copy(
+                        albums = albums,
+                    )
+                }
+            } catch (e: Exception) {
+                println(e)
```

Comment:
Aqui deberiamos poder actualizar algun estado en el viewmodel relacionado al error para mostrar un toast:

https://medium.com/smartherd/how-to-create-toast-message-in-jetpack-compose-android-6144c2749ae0

Reply:
Realizado 
![image](https://github.com/Santiago9913/MISW4203-Moviles/assets/142267769/02f755b9-0691-457e-98a5-590dbedc93c9)